src/python/grpcio_tests/tests/unit/_rpc_test_helpers.py

```diff
@@ -36,7 +36,7 @@
 _STREAM_STREAM = '/test/StreamStream'
 _STREAM_STREAM_NON_BLOCKING = '/test/StreamStreamNonBlocking'
 
-TIMEOUT_SHORT = datetime.timedelta(seconds=1).total_seconds()
+TIMEOUT_SHORT = datetime.timedelta(seconds=4).total_seconds()
```

Comment:
I actually only needed to make this change for a single test case. It's less about total duration and more about ordering. IIRC, the problem was that a timeout that our tests expected to be a FutureTimeout was actually an RPC timeout because, as you say, the RPC actually completed faster than the future coudl time out. I'll make the change in that test case rather than globally.